lib/kernel.js

```diff
@@ -101,7 +101,10 @@ module.exports = function(config, cb) {
 
   function stop() {
     logger.info('stopping operations');
-    _srv.stop();
+    _containers.stopAllServices(function(err) {
+      logger.error(err);
+    });
+    process.exit();
```

Comment:
you must put `process.exit` inside the callback, or you might lose some async behavior.